docs/sources/dashboards/build-dashboards/annotate-visualizations/index.md

```diff
@@ -19,96 +19,112 @@ you can get event description and event tags. The text field can include links t
 
 {{< figure src="/static/img/docs/v46/annotations.png" max-width="800px" >}}
 
-## Native annotations
+You can annotate panels in three ways:
```

Comment:
If " annotate " means creating a new annotation, this is misleading. There are 2 ways to create annotations, from the UI (panels) or the HTTP API. 
If we are talking about fetching annotations, I would say there is only one way: configuring annotations queries (built-in comes by default) in the dashboard settings.

Reply:
I mean it in the most general sense of the word, that you get annotations onto your panels in different ways. Maybe I could say something like this to clarify:

```suggestion
You can annotate panels in three ways. 

- Directly in the panel, using the [built-in annotations query](#built-in-query)
- Using the HTTP API
- Configure annotation queries in the dashboard settings

In the first two cases, you are creating new annotations, while in the last you're querying existing annotations from data sources. The built-in query annotation query also supports this.
```